Jenkinsfile

```diff
@@ -90,6 +90,19 @@ pipeline {
             }
         }
 
+        stage('Upload Android') {
+            when {
+                expression {
+                    TARGET_OS == 'All' || TARGET_OS == 'Android'
+                }
+            }
+            steps {
+                // upload apk artifact to folder(env.branch_name) in AWS S3 bucket (s3://symbol-mobile-builds/branches) 
+                // note that there is a s3 bucket retention policy deleting the files older than 60 days
```

Comment:
isn't 60 days a bit long?

Reply:
Previously checked with @cryptoBeliever, he said we might need the files for 2-3 months.

Reply:
are these published (release) builds or development builds?

Reply:
for feature branches: dev/testing purposes
dev/main branches: release candidates

btw the release is done manually(supervised).

Reply:
Asked @cryptoBeliever, he thinks that we can reduce it to 4 weeks (or 3 weeks if we push it ), good enough?